pkg/utils/utils.go

```diff
@@ -284,6 +284,51 @@ func CleanCachedNetConf(cRefPath string) error {
 	return nil
 }
 
+/*
+	SetVFEffectiveMAC will try to set the mac address on a specific VF interface
```

Comment:
indentation looks odd for the first line. should it be just a single space ?

Reply:
when you use `/*` for a function title it put the first line like that :)
I check it to `//`